src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunCrossgen.cs

```diff
@@ -201,7 +215,7 @@ public void It_does_not_support_cross_platform_readytorun_compilation(string tar
 
             var publishCommand = new PublishCommand(Log, Path.Combine(testProjectInstance.Path, testProject.Name));
 
-            publishCommand.Execute("/v:n")
+            publishCommand.Execute()
```

Comment:
Do you know why all the commands had "/v:n" in them?  Were any of them checking for output that might only be in included at normal verbosity?

Reply:
I was actually going to ask that, but there didn't seem to be any reason since this the others didn't check the output at all and this one was checking for an error.

Reply:
Probably a forgotten switch from the time I was debugging the test failures...